src/commands/config.py

```
@@ -0,0 +1,194 @@
+    # Check local guild permissions/owner
+    try:
+        if (
+            getattr(interaction.user, "guild_permissions", None)
+            and interaction.user.guild_permissions.manage_guild
+        ):
+            return True
+        if interaction.guild and interaction.user == interaction.guild.owner:
+            return True
+    except Exception:
+        # Be conservative and fall through to admin check
+        pass
```

Ruff (0.15.5)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)